langchain/embeddings/sagemaker_endpoint.py

```diff
@@ -71,17 +71,17 @@ class SagemakerEndpointEmbeddings(BaseModel, Embeddings):
     """
      Example:
         .. code-block:: python
-        
+
         from langchain.llms.sagemaker_endpoint import ContentHandlerBase
 
         class ContentHandler(ContentHandlerBase):
                 content_type = "application/json"
                 accepts = "application/json"
 
-                def transform_input(self, prompt: str, model_kwargs: Dict) -> bytes:
-                    input_str = json.dumps({prompt: prompt, **model_kwargs})
+                def transform_input(self, prompts: List[str], model_kwargs: Dict) -> bytes:
```

Comment:
The abstract class was updated to take `Union[str, List[str]]`, so we'll need to reflect that here. Thanks for the change!